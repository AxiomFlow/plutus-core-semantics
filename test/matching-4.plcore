(module Foo (imported) (exported () ()) (local () ())
  (define result
    (case (con Nat.Foo 3)
          (cl Nat.NotThisOne ()    -1)
          (cl Nat.NotThisTwo ()    -2)
          (cl Nat.NotThisThree (y) -3)
          (cl Nat.NotThisFour  (y)  -4)
          (cl Nat.Foo (x) (builtin multiplyInt (x 30)))
          (cl Nat.NotThisFive ()  -6))))
