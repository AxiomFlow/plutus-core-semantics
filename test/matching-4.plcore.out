<T> <k> . </k> <env> ( Foo . result ) |-> 90 </env> <mod> Foo </mod> </T>
